lib/index.js

```diff
@@ -68,6 +70,22 @@ exports.global = window;
 
 Segment.prototype.initialize = function() {
   var self = this;
+
+  if (this.options.retryQueue) {
+    this._lsqueue = new Queue('segmentio', function(item, done) {
+      // Update the sentAt time each retry so the tracking-api doesn't interperet a time skew
+      item.sentAt = new Date();
+      // send
+      send(item.url, item.msg, item.headers, function(err, res) {
```

Comment:
There's no longer a 1:1 relationship between a `analytics.*` call and a network request, which is how people check to make sure it was actually sent.

If we are sending the messages immediately on call, there will always only be 1. The only way to batch is by holding messages in memory for a period after the initial call. The library this is based on is written to not batch, because we didn't want to batch, so this isn't really an issue that could be handled here anyways.